.github/workflows/ci.yml

```diff
@@ -258,6 +258,11 @@ jobs:
         timeout-minutes: 5
         run: |
           make RELEASE=true UNINSTALL=noclean smoke-test-tls-root
+      - name: Test examples
+        timeout-minutes: 10
+        run: |
+          examples/tests/test-simple.sh --release
+          examples/tests/test-echo.sh --release
```

Comment:
why is there duplicate run after smoke test?

Reply:
One is Mac and one is Linux